spec/orm/activerecord_spec.rb

```diff
@@ -1195,6 +1197,14 @@ def monkey
     end
 
     describe "#remote_images_urls=" do
+      before do
+        sham_rack_app = ShamRack.at('www.example.com').stub
+        sham_rack_app.register_resource('/test.jpg', File.read(file_path('test.jpg')), 'images/jpg')
```

Comment:
Opened at #1829 :+1: 
I'll rebase this one depending on which direction we are going.